querier_selects.go

```diff
@@ -42,8 +42,12 @@ func (q *Querier) selectQuery(view View, tail string, limit1 bool) string {
 		query += " TOP 1"
 	}
 
-	return fmt.Sprintf("%s %s FROM %s %s",
-		query, strings.Join(q.QualifiedColumns(view), ", "), q.QualifiedView(view), tail)
+	from := q.QualifiedView(view)
+	if q.qualifiedViewName != "" {
+		from = q.qualifiedViewName + " AS " + from
+	}
+
+	return fmt.Sprintf("%s %s FROM %s %s", query, strings.Join(q.QualifiedColumns(view), ", "), from, tail)
```

Comment:
Here we only change table name in `FROM` clause and alias it to the old name. That should do the trick for all `SELECT`s. @bitxel, please check it out.

Reply:
understand, it's a neat way that only change the view name when making SQL, 
but it may lose the schema name, so it's caller's responsibility to add and quote the schema name